controller/array_action/array_mediator_svc.py

```diff
@@ -502,15 +491,20 @@ def _delete_fcmap(self, fcmap_id, force):
             self.client.svctask.rmfcmap(object_id=fcmap_id, force=force)
         except (svc_errors.CommandExecutionError, CLIFailureError) as ex:
             if not is_warning_message(ex.my_message):
-                logger.warning("Failed to delete fcmap '{0}': {1}".format(fcmap_id, ex))
+                logger.exception("Failed to delete fcmap '{0}': {1}".format(fcmap_id, ex))
```

Comment:
`logger.error` is sufficient, no need to log the traceback (same for the other logger calls)

Reply:
+1

Reply:
done